docs/core/profiles.md

```diff
@@ -151,96 +153,112 @@ can be influenced by the different fields of such a custom model.
+  road_class == MOTORWAY: 0.5
+```
+
+You can also change the speed factor for multiple road classes like this
 ```yaml
 speed_factor:
-  road_class:
-    motorway: 0.5
+  road_class == MOTORWAY: 0.5
+  road_class == PRIMARY || road_class == TERTIARY: 0.7
 ```
 
-You can also setup speed factors for multiple road classes like this
+Please note the OR operator `||` that says that if one of the expressions is true the value 0.7 is used.
+
+or use multiple categories to influence the speed factor
 ```yaml
 speed_factor:
-  road_class: {motorway: 0.5, primary: 0.7, tertiary: 0.9}
+  road_class == MOTORWAY: 0.5
+  road_environment == TUNNEL: 0.8
 ```
 
-and use multiple categories to influence the speed factor
+You can use only two operators `==` (equality) and `!=` inequality.
+
+If a road segment matches multiple expressions the speed factor values will be multiplied. For example, here the speed factor of
+a road segment that has `road_class==MOTORWAY` will be `0.5`, the speed factor of a road segment that additionally has
+`road_environment==TUNNEL` will be `0.4` and the speed factor of a road segment that has `road_class==SECONDARY` and
+`road_environment==TUNNEL` will be `0.8`. You can avoid this by using the "first_match" directive:
+
 ```yaml
 speed_factor:
-  road_class: {motorway: 0.5}
-  road_environment: {tunnel: 0.8}
+  first_match:
+    road_class == MOTORWAY: 0.5
+    road_environment == TUNNEL: 0.8
 ```
 
-If an edge matches multiple rules the speed factor values will be multiplied. For example, here the speed factor of 
-a road segment that has `road_class=motorway` will be `0.5`, the speed factor of a road segment that additionally has 
-`road_environment=tunnel` will be `0.4` and the speed factor of a road segment that has `road_class=secondary` and 
-`road_environment=tunnel` will be `0.8`.
+Now even if a road segment fulfills both conditions only the first will be used and no multiplication happens. If you
+know a programming language than the previous statements are "if" clauses and expressions under "first_match" are
+"if-then-else-if" clauses.
 
-Instead of setting the speed factors for certain values you can instead set the speed factors for all *other* values using
-as special key (`"*"`), like this:
+Instead of setting the speed factors for certain values you can instead set the speed factors for all *other* values
+using the unconditional expression `true`:
 ```yaml
-speed_factor: 
-  road_class: {"*": 0.5}
-  road_environment: {tunnel: 0.8, "*": 0.6}
+speed_factor:
+  true: 0.5
+  first_match:
+    road_environment == TUNNEL: 0.8
+    true: 0.6
 ```
 
-So in this example we set a speed factor of `0.5` regardless of the `road_class` and all `road_environment` values yield
-a speed factor of `0.6` *except* `tunnel` which gets a speed factor of `0.8`. And as mentioned above for edges that match
-multiple of these rules the different factors get multiplied.
+This is usually used in the `first_match' directive.
```

Comment:
I find this hard to understand. First you introduce `first_match` as something that can be used to avoid the multiplication rule, but in this example we multiply everything with `0.5` again. Would you not rather set 

```yaml
first_match:
    road_environment == TUNNEL: 0.4
    true: 0.3
```
?
Maybe we can find a better example where it makes sense to combine the default expressions with `first_match`?

Reply:
Also I wonder why we only have `first_match`? What if we need multiple explicit rules, like 

```yaml
first_match:
    road_environment == TUNNEL: 0.4
first_match:
    surface == GRAVEL: 0.7
```

Reply:
Oh, you are right the example in profiles.md is ugly. Better would be something like:

```yaml
speed_factor:
  road_class == PRIMARY: 0.5
  first_match:
    road_environment == TUNNEL: 0.8
    true: 0.6
```

> Also I wonder why we only have first_match? What if we need multiple explicit rules, like

What do you have for a use case in mind? Your specific example could be converted into:

```yml
road_environment == TUNNEL: 0.4
surface == GRAVEL: 0.7
```

Multiple first_matches are possible via:

```
first_match_1:
    road_environment == TUNNEL: 0.4
    some_other_expression: 0.3
first_match_2:
    surface == GRAVEL: 0.7
    some_other_expression_2: 0.3
```